src/actions/loginAction.js

```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { LOGIN_USER_SUCCESS, LOGIN_USER_FAILED } from '../reducers/auth/login/actionTypes';
+import history from '../utils/history';
+
+const apiUrl = 'https://ah-backend-kronos-staging.herokuapp.com/api/users/login/';
+
+
+const loginUserAction = userData => dispatch => axios.post(apiUrl, userData)
+  .then((response) => {
+    dispatch({
+      type: LOGIN_USER_SUCCESS,
+      payload: response.data,
+    });
+    toast.dismiss();
+    toast.success(`Welcome ${response.data.user.username}. Login Successful`, {
+      containerId: 'A',
+      position: toast.POSITION.TOP_RIGHT,
+      autoClose: false,
+      hideProgressBar: false,
+    });
```

Comment:
We implemented a re-usable toast in the utils folder so this is not necessary.

Reply:
Resolved.